modules/netstat/netstat.go

```diff
@@ -84,6 +86,11 @@ type statistics struct {
 	Totalhits float64 `json:"totalhits"`
 }
 
+// Indicates module should use namespace resolution mode, which changes
+// the behavior on Linux; optional right now but may be the default in
+// the future.
+var namespaceMode bool
```

Comment:
it makes sense to me for that to be default since we ignore if namespace cant be read.
this is only optional because its new i guess right?